lib/XML/LibXML/CStructs.pm

```diff
@@ -113,6 +119,27 @@ my class xmlError is export(:types) {
     has OpaquePointer $.node; # the node in the tree
 }
 
+my class xmlElement is export(:types) {
```

Comment:
Every structure we use... if the P6 class definition is incorrect, we wont allocate enough bytes and the C lib might access unallocated memory and will explode. if our class definition is bigger byte-wise we will access mem taht the C lib might not have allocated.